pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -898,14 +899,35 @@ func possibleGuestSize(disk api.Disk) (int64, bool) {
 		return 0, false
 	}
 
-	size := int64((1 - filesystemOverhead) * float64(preferredSize))
+	usableSize, err := getUsableDiskSize(getSourceFile(disk))
+	if err != nil {
+		log.DefaultLogger().Reason(err).Error("Failed to get total usable space, using disk capacity instead")
+		usableSize = *disk.Capacity
+	}
+
+	size := int64((1 - filesystemOverhead) * float64(usableSize))
```

Comment:
This will generate a size that is not aligned, this will cause issues with qemu, please keep the AlignImageSizeTo1MiB call.

Reply:
The size is aligned below this line. The test are failing though and I'm not sure if it's just due to the block stat, so I'm still investigating.

Reply:
Wait nevermind you are calling it.